modular_skyrat/modules/colony_fabricator/code/tools/tools.dm

```diff
@@ -26,7 +26,7 @@
 	hitsound = 'sound/items/drill_hit.ogg'
 	usesound = 'sound/items/drill_use.ogg'
 	w_class = WEIGHT_CLASS_SMALL
-	toolspeed = 1
+	toolspeed = 1.2
```

Comment:
Just to make sure, this one's description explicitly says "Compared to other power drills these tend to be \
		<b>not much quicker than unpowered tools</b>." So it SHOULD still be quicker than unpowered ones.

Reply:
Just leave it at 1 toolspeed. You don't even gain much from it being three in one compared to power tools, considering that you still need to carry a separate crowbar.

Reply:
if they're 1 toolspeed, they should be normal sized like the other powertools; or powertools should be made small sized, because essentially-power-tools but only one is normalsized (crowbar) is a bit off pilled with the standard currently set, when you can do stuff like (image), so I decided it'd be better to just slow them down a bit so they aren't always questionably a storage upgrade to powertools for non-engineers (minus the prybar)
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/62520989/a3f7533a-c549-4caa-a2d3-00d169eeedbd)

Reply:
Make the driver normal sized in that case, I thought I did already. The convenience of multiple in one NOTHING if you're staring at action bars longer than just having the tools split.

Reply:
> if they're 1 toolspeed, they should be normal sized like the other powertools; or powertools should be made small sized, because essentially-power-tools but only one is normalsized (crowbar) is a bit off pilled with the standard currently set, when you can do stuff like (image), so I decided it'd be better to just slow them down a bit so they aren't always questionably a storage upgrade to powertools for non-engineers (minus the prybar) ![image](https://user-images.githubusercontent.com/62520989/277208507-a3f7533a-c549-4caa-a2d3-00d169eeedbd.png)

Powertools should probably be made small sized, yes.

Reply:
They aren't because it would completely invalidate the tool belt as a belt, you either choose to have a belt and faster tools, a box with slower tools, or half your backpack space to have fast tools